README.md

```diff
@@ -1 +1,155 @@
-# FIRST-CAPSTONE-PROJECT
\ No newline at end of file
+# HTML CSS and javascript capstone project
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    -[Tech Stack](#tech-stack)
+- [🚀 Live Demo](#live-demo) 
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+-- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [HTML CSS and javascript capstone project] <a name="HTML CSS and javascript capstone project"></a>
+
+
+**[HTML CSS and javascript capstone project]** is a HTML, CSS and JavaScript project that teaches how to Use JavaScript to manipulate DOM elements, Use JavaScript events, Use objects to store and access data, use Flexbox to place elements in the page, use media query to build a fully responsive personal portfolio site.
+
+## 🛠 Built With  <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+Click here to see a live demo of this project online: <li><a href="https://abdelazizsoliman.github.io/FIRST-CAPSTONE-PROJECT//">Live Demo</a></li>
+
+Click here to see a video presentation on LOOM: <li><a href="https://www.loom.com/share/783a87ff2475407aac8010db66b91bed">Show video</a></li>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your "details popup window" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+-A Git hub account
+-Git bash
+-Node JS
+-Visual Studio Code as your code editor
+-Figma account
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone[ (https://github.com/AbdelazizSoliman/FIRST-CAPSTONE-PROJECT.git)]
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdelaziz Soliman**
+
+- GitHub: [@abdelazizsoliman](https://github.com/abdelazizsoliman)
+- Twitter: [@Abdelaz21958317](https://twitter.com/Abdelaz21958317)
+- LinkedIn: [Abdelaziz Soliman](https://www.linkedin.com/in/abdelaziz-soliman-16989b89/)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [optional] kindly remove this comment as it was just meant to help guide you on the information to add to that section. removing it keeps your readme clean